genie/database_to_staging.py

```diff
@@ -152,21 +152,23 @@ def _redact_year(df_col):
 
 # TODO: Add to transform.py
 def _to_redact_difference(df_col_year1, df_col_year2):
-    """Determine if difference between year2 and year1 is > 89
+    """Determine if difference between year2 and year1 is > 89 or is < 18
 
     Args:
         df_col_year1: Dataframe column/pandas.Series of a year column
         df_col_year2: Dataframe column/pandas.Series of a year column
 
     Returns:
-        pandas.Series: to redact boolean vector
+        tuple: pandas.Series: to redact boolean vector
+               pandas.Series: to redact pediatric boolean vector
 
     """
     # Add in errors='coerce' to turn strings into NaN
     year1 = pd.to_numeric(df_col_year1, errors="coerce")
     year2 = pd.to_numeric(df_col_year2, errors="coerce")
     to_redact = year2 - year1 > 89
-    return to_redact
+    to_redact_ped = year2 - year1 < 18
```

Comment:
Ah! Don't redact <18. This ticket was only for >89 because we have plans on removing the redaction plan for <18.

I don't know if that complicates things if we keep current behavior for <18

Reply:
ahh I thought we want to redact ped anyway but just don't know how to label it. Got it. I will remove it